x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/SqlIllegalArgumentException.java

```diff
@@ -1,9 +1,9 @@
-package org.elasticsearch.xpack.eql;
 /*
  * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
  * or more contributor license agreements. Licensed under the Elastic License;
  * you may not use this file except in compliance with the Elastic License.
  */
+package org.elasticsearch.xpack.eql;
 
 public class SqlIllegalArgumentException extends EqlServerException {
```

Comment:
Should this be `EqlIllegalArgumentException`?

Reply:
Not sure how this file got it - there's already `EqlIllegalArgumentException `.
Thanks for noticing it - I'll remove it.